bin/commands/indexDump.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2018 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-console */
+
+const
+  fs = require('fs'),
+  ColorOutput = require('./colorOutput'),
+  path = require('path'),
+  getSdk = require('./getSdk');
+
+function mkdirp (fullPath) {
```

Comment:
(nitpicking) can you add a TODO comment asking to remove that function for when our min. Node.js version will be at least Node.js 10?
(starting from node 10, we can use the option `recursive: true` with [fs.mkdir](https://devdocs.io/node~10_lts/fs#fs_fs_mkdir_path_options_callback))